### PR TITLE
[lgwebos] Increase limit to 5MB for Websocket text message

### PR DESCRIPTION
**Pull request by lolodomo**
A user reported an error with size 4 316 809 while previous max was 4 194 304.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by lsiepel — APPROVED** [review]
LGTM, maybe wait for @sprehn to approve before merge.

**Comment by jlaur** [comment]
> LGTM, maybe wait for @sprehn to approve before merge.

Last activity in 2021, so this could be a long wait. 😉 Since it's a simple non-controversial fix, let's merge.

**Pull request merged**
